consul-exports.ctmpl

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+{{scratch.Set "env" (env "APP_ENV") -}}
+{{$appname := env "APP_NAME" -}}
```

Comment:
@d4v3y0rk it's [already there](https://github.com/articulate/docker-consul-template-bootstrap/blob/drop-vault-req/export-vault.ctmpl#L3). Am I misunderstanding?